doc/nrf/releases_and_maturity/migration/nRF54H20_migration_2.7/migration_guide_2.4.99-cs3_to_2.7_application.rst

```diff
@@ -74,6 +74,30 @@ General
 
      The previously used SOC1-based board files have been removed.
 
+CoreMark
+--------
+
+Several changes has been made to migrate the CoreMark sample from the nRF54 customer sampling release v2.4.99-cs3 to |NCS| v2.7.0:
```

Comment:
```suggestion
Several changes have been made from the nRF54 customer sampling release v2.4.99-cs3:
```

Reply:
I actually don't describe all changes that have been made. I just describe changes related to migration.

Reply:
Ok, didn't understand what you wanted to say :)

Still this sentence is somehow bad. The problem is with phase \"migration". In this guide as far as my understanding goes, it means the actions needed by customer to preform. But it can also mean work conducted by us while moving it from one repository to another. I would avoid using it in this guide in second meaning.

Reply:
I use it in the 2nd meaning, but if you would like to phrase it differently then propose how would you see it,

Reply:
or we can wait for tech writers

Reply:
Didn't understood correctly it is good as it is.

Reply:
I've listed the changes that needed to be done to port app from nrf-next to main nrf. I didn't mentio the changes that were made due to other reasons.